wazimap_ng/datasets/admin/indicator_data_admin.py

```diff
@@ -38,8 +49,80 @@ def parent(self, obj):
     list_filter = (filters.IndicatorFilter,)
 
     search_fields = ["geography__name"]
+    
+    class Media:
+        css = {
+             'all': ('/static/css/admin-custom.css',)
+        }
 
     def get_readonly_fields(self, request, obj=None):
         if obj: # editing an existing object
             return ("geography", "indicator") + self.readonly_fields
-        return self.readonly_fields
\ No newline at end of file
+        return self.readonly_fields
+    
+    def get_urls(self):
+        urls = super(IndicatorDataAdmin, self).get_urls()
+        my_urls = [
+            path("upload/", self.upload_indicator_director),
+        ]
+        return my_urls + urls
+
+    def upload_indicator_director(self, request):
+
+        if request.method == 'POST':
+            form = IndicatorDirectorForm(request.POST, request.FILES)
+            if form.is_valid():
+                indicator_director = request.FILES["indicator_director"]
+                dataset = form.cleaned_data["dataset"]
+                logger.debug(f" Uploaded Indicator director file: {indicator_director}")
+                
+                indicator_indicator_json = indicator_director.read()
```

Comment:
```suggestion
                indicator_director_json = indicator_director.read()
```
Or I'm missing something?